spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  # rubocop:disable all
+  factory :user do
+  end
+  # rubocop:enable all
```

Comment:
RuboCopの警告があったようですが、おそらくempty blockみたいな感じで怒られますかね、、？

以下のようにすると警告消えるかなと思いましたが、どうでしょうか？
```suggestion
  factory :user
```

Reply:
ありがとうございます！
修正案の通り修正したら、警告消えましたのでこちらに書き換えています！